src/ShellyEssentials/EventListener.php

```diff
@@ -58,7 +58,9 @@ public function onEntityDamage(EntityDamageEvent $event) : void{
 	}
 
 	public function onPlayerLogin(PlayerLoginEvent $event) : void{
-		$event->getPlayer()->teleport(API::getMainInstance()->getServer()->getDefaultLevel()->getSafeSpawn());
+        $player = $event->getPlayer();
```

Comment:
Don't need this variable, just use $event->getPlayer() since you only use it twice in this whole function lol